src/document.h

```diff
@@ -70,6 +70,9 @@ typedef struct GeanyFilePrefs
 GeanyFilePrefs;
 
 
+#define GEANY_TYPE_DOCUMENT (document_get_type())
+GType document_get_type (void);
```

Comment:
interesting question.  But it probably doesn't matter so much as it's not in docs.